### PR TITLE
Improve helm orchestration config

### PR DESCRIPTION
**Pull request by iffyuva**
Small improvements in helm config and allow status checker to follow redirects.

### CODE REVIEW

File: orchestration/helm/templates/configmap.yaml [hunk 1]
```diff
@@ -3,8 +3,9 @@ kind: ConfigMap
 metadata:
   name: {{ template "dockup.fullname" . }}
 data:
+  MIX_ENV: prod
   DOCKUP_WORKDIR: "/workdir"
-  DOCKUP_DOMAIN: "{{ .Values.ingress.host }}"
```
Comment (emilsoman) [inline orchestration/helm/templates/configmap.yaml]: Can we explicitely use a value like `Values.dockup.domain` in Values file , and also reuse it in the ingress section?

File: orchestration/helm/templates/ingress.yaml [hunk 2]
```diff
@@ -26,11 +26,13 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-    - host: {{ .Values.ingress.host }}
+  {{- range .Values.ingress.hosts }}
```
Comment (emilsoman) [inline orchestration/helm/templates/ingress.yaml]: When do we have multiple hosts?

File: orchestration/helm/templates/configmap.yaml (hunk 1 above)
Comment (iffyuva) [reply orchestration/helm/templates/configmap.yaml]: @emilsoman its part of follow up PR

File: orchestration/helm/templates/ingress.yaml (hunk 2 above)
Comment (iffyuva) [reply orchestration/helm/templates/ingress.yaml]: I reverted this code to boilerplate generated by `helm create` method

**Pull request merged**